CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# 2.0.0
+
+Notes on migration of 1.1.0 data:
```

Comment:
Add more text here, for example:

Version 2 of the rsd requires changes to the datatabase. Below are the steps to migrate a rsd instance of version 1 to version 2.